cypress-tests/cypress/e2e/ConnectorTest/00011-CreateMultiuseMandate.cy.js

```diff
@@ -7,7 +7,7 @@ import State from "../../utils/State";
 
 let globalState;
 
-describe("Card - MultiUse Mandates flow test", () => {
+describe("Card - SingleUse Mandates flow test", () => {
```

Comment:
isn't these test cases are related to multi use? please update the `describe` of the test case